torch/csrc/inductor/aoti_torch/shim_common.cpp

```diff
@@ -11,6 +11,7 @@
 #include <cstdio>
 #include <iostream>
 #include <memory>
+#include <ATen/native/BinaryOps.h>
```

Comment:
not sure if this include is OK, or if I should make it part of `AT_PER_OPERATOR_HEADERS`?